### PR TITLE
Maintainers profile page

### PR DESCRIPTION
**Pull request by maneesha**
This updates the Maintainers profile page so that only currently active Maintainers are listed.  It adds a Maintainer Alumni page to list badged Maintainers without an active role.

### CODE REVIEW

**Comment by maneesha** [comment]
Would like review from someone on Curriculum team (@tobyhodges @ErinBecker @zkamvar @MCMaurer )

File: pages/maintainer_alumni.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+{% include lazyload.html %}
+
+<section id="team" class="bg-light-gray">
+<div class="container">
+
+{% assign role = site.data.communities | where: "name", "Maintainer Alumni"  %}
+{% include community_profile_header.md %}
+
+<p>Check out our active Maintainer community <a href="{% link pages/maintainers.html %}">here</a>.</p>
```
Comment (zkamvar) [inline pages/maintainer_alumni.html]: Give context for the link.

```suggestion
<p>Check out our <a href="{% link pages/maintainers.html %}">active Maintainer community</a>.</p>
```

**Review by zkamvar — APPROVED** [review]
Looks good to me with one request for accessibility

**Comment by zkamvar** [comment]
There is, however one thing I notice in the maintainer alumni page: there are doppelgängers.

screenshot of maintainer alumni page showing a pair doubled maintainer profiles amidst the non-doubled maintainers

**Comment by maneesha** [comment]
Thanks for catching that @zkamvar - This is because we are now counting **roles**, not just badges, and some people are maintainers for more than one lesson.  I'll update the query to exclude duplicate names.

**Comment by maneesha** [comment]
The issue with duplicates has been fixed (you may need to refresh your browser to see) so I will merge this.  Please open an issue if anything else looks off.

**Comment by ErinBecker** [comment]
Thank you @maneesha and @zkamvar!!

**Pull request merged**